indico/modules/events/templates/footer.html

```diff
@@ -1,5 +1,24 @@
 {% extends 'footer.html' %}
 
+{% block footer_links_extra %}
+    {% if privacy_url %}
+        <li>
+            <a href="{{ privacy_url }}">{% trans %}Event Privacy Policy{% endtrans %}</a>
```

Comment:
s/Policy/Notice/ (and let's keep the capitalization consistent between here and below)

Reply:
oops, missed that one. thanks!